test/e2e/common.go

```diff
@@ -0,0 +1,154 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+
+	bmov1 "github.com/metal3-io/baremetal-operator/apis/metal3.io/v1alpha1"
```

Comment:
`bmov1` is a bit confusing name to give for an alpha api, may be `metal3api` as the rest of the BMO codebase (mostly)

Reply:
I don't see how it is confusing, but ok if we use `metal3api` in other places I can use the same here. For the record we use `infrav1` for CAPM3 v1beta1, `ipamv1` for IPAM v1alpha1 and `clusterv1` for CAPI v1beta1 in CAPM3. Seems `bmov1` would be perfectly following the trend and easy to understand :wink:

Reply:
Changed to `metal3api`.

Reply:
Thanks for changing to `metal3api`. When it comes to `infrav1` for CAPM3, it was changed 'infrav1' when we switched to CAPM3 v1beta1, before that for CAPM3 `v1alphaX` APIs we were using `infrav1alpha5`, for IPAM it was 'ipamv1' ever since we initialized the repo :confused: . I thought theres would be some best practices around it but couldn't find one. What I found is namings are `v1` mostly when APIs are `v1` or `v1betaX`, for alpha APIs naming includes `v1alpha1` or then some completely other name like `metal3api`.